samples/zigbee/ncp/README.rst

```diff
@@ -94,6 +94,9 @@ The USB device VID and PID are configured by the sample's Kconfig file.
 .. note::
     The USB is used as the default NCP communication channel when using the nRF52840 Dongle.
 
+You can configure :ref: Zigbee stack logs`<zigbee_ug_logging_stack_logs>` to be printed in binary form over USB using independent serial port.
+Once configured, no additional code is required.
```

Comment:
Code required where? What does this mean specifically?

Reply:
For NCP sample, this can be configured purely by setting correct Kconfig options - no changes in code are required.